tests/Microsoft.Data.DataFrame.Tests/DataFrameTableTests.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.Data;
+using Xunit;
+
+namespace Microsoft.Data.Tests
+{
+    public class DataFrameTableTests
+    {
+        static public DataFrame MakeTestTableWithTwoColumns(int length)
+        {
+            BaseDataFrameColumn dataFrameColumn1 = new PrimitiveDataFrameColumn<int>("Int1", Enumerable.Range(0, length).Select(x => x));
```

Comment:
PrimitiveDataFrameColumn feels very verbose for such a common thing. It would be good to have some shortcuts to create them, like some static functions that could at least infer the generic type and maybe just be called Column()